TileStache/Goodies/VecTiles/sort.py

```diff
@@ -76,8 +77,16 @@ def landuse(features, zoom):
     return _sort_by_area_then_id(features)
 
 
+def _place_key_desc(feature):
+    sort_key = _by_population(feature), _by_area(feature)
```

Comment:
Yea, that's right. For features that don't have `population`, it'll use `None` there which will be a no-op for those features, ie not change the sort order.

The order of the sort keys / sorts themselves is important though to define which features of each "type" come first when there's a mix. Eg if WOF features should come before OSM features in the layer.